docs/source/intro/configure.rst

```diff
@@ -154,4 +154,19 @@ it into the format above, you can use
 
 This constructs a :setting:`DATABASES` setting equivalent to the first example.
 
+.. _configuring-database-routers-setting:
+
+Configuring the ``DATABASE_ROUTERS`` setting
+============================================
+
+If you intend to use :doc:`embedded models </topics/embedded-models>`, you must
+configure the :setting:`DATABASE_ROUTERS` setting so that a collection for
+these models isn't created and so that embedded models won't be treated as
+normal models by :djadmin:`dumpdata`::
+
+    DATABASE_ROUTERS = ["django_mongodb_backend.routers.MongoRouter"]
```

Comment:
@aclark4life 
Can you make a change to the django-mongodb-project template to have this be a default setting?

Reply:
Will do

Reply:
https://github.com/mongodb-labs/django-mongodb-project/pull/10